templates/base.html

```diff
@@ -4,6 +4,7 @@
   <meta c="UTF-8"/>
   <title>F1 League</title>
   <link rel="stylesheet" href="/static/main.css">
+  {% block head %}{% endblock %}
```

Comment:
Hmm, so I went with [`DOMContentLoaded`](https://github.com/hmac/f1/pull/10/files#diff-7a66652fd49c10322647723e0d8fdb0aR57) in this case, since it's so light it's likely to make little difference to page render time. However, were we to start bundling up libraries with the js then yeah a shift to the body might be better.

Tbh actually, `defer` is probably the best option here. I doubt we care about IE9...